delivery_carrier_category/README.rst

```diff
@@ -0,0 +1,74 @@
+=================
+Delivery Category
+=================
```

Comment:
no need to create README.rst instead you need to add a folder readme with sections, bot will regenerate Reame.rst based on sections after merge 
https://github.com/OCA/maintainer-tools/tree/master/template/module/readme